packages/components/src/calendar/styles.module.scss

```diff
@@ -2,22 +2,22 @@
 @use "../styles/typography";
 
 /* Root of the component. */
-.a8c-components-calendar {
+.calendar {
 	// TODO:
 	// - move`font-family` rule (and "../styles/typography" import) out of component
 	// - add `font-family` rule to Storybook's global styles
 	// - when Theme is ready and includes type styles, add back to component
 	font-family: typography.$a8c-font-family-sans;
 
 	// Internal variables
-	--a8c-components-calendar-outline-focus: 1.5px solid var(--color-accent);
-	--a8c-components-calendar-button-height: 2rem;
-	--a8c-components-calendar-button-width: 2rem;
-	--a8c-components-calendar-nav-height: 2rem;
+	--calendar-outline-focus: 1.5px solid var(--color-accent);
```

Comment:
I removed the `a8c-components-` prefix from local CSS variables. Is should be safe (despite CSS variable names not being mangled), but wanted to flag anyway

Reply:
That's an interesting topic: Should we be worried about naming collisions for CSS variables that won't be mangled by CSS modules? It seems relatively low risk here, but I could imagine something like `--button-*` potentially running into issues. Maybe the namespacing prefix is valuable, if even verbose? Maybe just `--a8c-` ?